tests/test_expectations.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2018 Carsten Blank
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for :mod:`pennylane_qiskit` expectation values
+"""
+import logging as log
+from pennylane import numpy as np
+
+from defaults import pennylane as qml, BaseTest, IBMQX_TOKEN
+from pennylane_qiskit import BasicAerQiskitDevice, AerQiskitDevice, IbmQQiskitDevice
+
+log.getLogger('defaults')
+
+
+class TestQVMBasic(BaseTest):
+    """Unit tests for the QVM simulator."""
+    # pylint: disable=protected-access
+
+    num_subsystems = 2
+    shots = 16 * 1024
```

Comment:
This leads to an error from qiskit saying that the number of shots exceed some limit. I changed this to 8*1024 and it worked.

Reply:
Oh that can actually be the case for the IBM Q backends. Very unfortunate though! Because many tests will fail because of very strong deviations from randomness. Any idea how to achieve both goals: satisfying constraints and failing by randomness?

Reply:
I chose to solve this by using 16*1024 shots for the local backends and 8*1024 for the IBMQ. Then it won't fail as often.... not so convincing though!

Reply:
Sorry, at the moment I have no idea how to do that but seems like the current settings work. Maybe we need to move the in-depth testing on the IBM device somewhere else? Like a dev-branch?

Reply:
I agree! The testing takes really really long...!!